lib/post_creator.rb

```diff
@@ -507,6 +507,9 @@ def create_topic
     if @topic && @topic.category && @topic.category.all_topics_wiki
       @post.wiki = true
     end
+    if @topic && @topic.category && @topic.category.default_slow_mode_seconds
+      @topic.update!(slow_mode_seconds: @topic.category.default_slow_mode_seconds)
```

Comment:
Let's update slow mode like how we do it for autoclose: https://github.com/discourse/discourse/blob/f26acb4b634e148bf0c1f1dfc158859329e4beae/app/models/topic.rb#L336

Reply:
I wasn't sure if you meant "how we do it for autoclose" in terms of "setting a topic timer", or just "putting it in `models/topic.rb` as a separate method" – I did the latter because a) it was easier 😅 and b) as per the original commit message, I can't imagine the use-case for having time-limited auto "slow mode" in a category.